CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * Add new method `MustParsePackage` to create new packages, running more validations. [#1333](https://github.com/elastic/package-registry/pull/1333)
 * Allow to customize settings related to SQL storage indexer (technical preview). [#1334](https://github.com/elastic/package-registry/pull/1334) [#1337](https://github.coim/elastic/package-registry/pull/1337)
 * Cleanup SQL storage indexer backup database only when a new index version is downloaded (technical preview). [#1337](https://github.coim/elastic/package-registry/pull/1337)
+* Allow to define deployment modes in each input of the policy templates. [#1345](https://github.coim/elastic/package-registry/pull/1345)
```

Comment:
Nit. The change that allows defining deployment modes is in the Spec, maybe here we should say that deployment modes are included in responses.
```suggestion
* Include deployment modes in responses when they are defined at inputs of the policy templates. [#1345](https://github.coim/elastic/package-registry/pull/1345)
```